### ISSUE TITLE
Everything looks super small on Google Pixel 3a XL

### ISSUE THREAD

**Issue by andywer** [satoshipay/solar#866]
Seems like the screen has a very high resolution, but for some reason the web view doesn't adapt to it.

**Comment by Marcel-Ebert** [comment]
I'm not 100% sure if I am able to reproduce this.
I used an Android Emulator for the Pixel 3a XL device running API 28 and it looks normal on start.
By changing display size and font size in system settings I was able to make it look a bit smaller but I am not sure if this 'super small'. 
@andywer did it look like on the screenshots below or even smaller?

![Screenshot_1574954320](https://user-images.githubusercontent.com/6690623/69817404-d1f1c380-11fa-11ea-8c80-521d52d97caa.png)
![Screenshot_1574954323](https://user-images.githubusercontent.com/6690623/69817405-d1f1c380-11fa-11ea-9860-0e1c5673db32.png)
![Screenshot_1574954341](https://user-images.githubusercontent.com/6690623/69817406-d1f1c380-11fa-11ea-8aab-8737ef9c364a.png)
